packages/patternfly-4/react-core/src/layouts/Grid/Grid.js

```diff
@@ -25,6 +25,8 @@ const propTypes = {
   lg: gridItemSpanValueShape,
   /** the number of columns all grid items should span on a xLarge device */
   xl: gridItemSpanValueShape,
+  /** the number of columns all grid items should span on a 2xLarge device */
+  xl2: gridItemSpanValueShape,
```

Comment:
should we call it `2xl` or is there a reason it's `xl2` here?

Reply:
I had to name it xl2 to meet JSX attribute name requirements. It won't allow a prop to start with a number. :(

Reply:
Maybe `'2xl': gridItemSpanValueShape` will make JSX happy?

Reply:
That works for defining the prop, but it doesn't work when actually creating a `<GridItem 2xl={2}>`